cmd/ncproxy/run.go

```diff
@@ -27,14 +28,42 @@ import (
 	"go.opencensus.io/plugin/ocgrpc"
 	"go.opencensus.io/trace"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 type nodeNetSvcConn struct {
 	client   nodenetsvc.NodeNetworkServiceClient
+	v0Client nodenetsvcV0.NodeNetworkServiceClient
 	addr     string
 	grpcConn *grpc.ClientConn
 }
 
+func (n *nodeNetSvcConn) ConfigureNetworking(ctx context.Context, req *nodenetsvc.ConfigureNetworkingRequest) (*nodenetsvc.ConfigureNetworkingResponse, error) {
```

Comment:
are we going to try the v1 and fall back on v0 every time configure networking is called? is `ConfigureNetworking` a call we anticipate will be called frequently?

Reply:
Yes. This call is made on the creation of every pod. Falling back allows us to have support for both v0 and v1 without needing to update the ncproxy binary or restart an already running ncproxy.

Reply:
oh, so the goal is to seemlessly switch mid operation, basically